task/jenkins/0.1/jenkins.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: jenkins
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: jenkins, build
+    tekton.dev/displayName: "jenkins operation"
+spec:
+  description: >-
+    The following task can be used to interact with the the Jenkins REST API.
+  workspaces:
+    - name: source
+      description: >-
+        The workspace which can be used to mount files which can be
+        send via the API to the Jenkins job.
+  results:
+    - name: build_number
+  params:
+    - name: JENKINS_HOST_URL
+      type: string
+      description: Server URL on which Jenkins is running
+    - name: JENKINS_SECRETS
+      type: string
+      description: Jenkins secret containing credentials
+      default: jenkins-credentials
+    - name: ARGS
+      type: array
+      description: The argument to send to the control script, see README of this task for details.
+      default: ["--help"]
+  steps:
+    - name: jenkins-control
+      image: registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
I don't think the UBI image has been designed to be "pinned" : 

see documentation here: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/building_running_and_managing_containers/using_red_hat_universal_base_images_standard_minimal_and_runtimes#using_minimal_red_hat_base_images

``` Red Hat intends for you to always use the latest version of the minimal images, which is implied by requesting ubi8/ubi-minimal or ubi8-minimal. Red Hat does not expect to support older versions of minimal images going forward. ```

Reply:
but we need to pin the version of the image...maybe we can use `8.2` for now because in future when we are going to include catlin in the CI it will error out this task as it is using the `latest` tag

Reply:
yeah `ubi:8.2` should work just fine :wink:

Reply:
So yeah that didn't work just fine... the tags are a bit all over the place and found out those by pure luck : 

for ubi it is : 

```registry.access.redhat.com/ubi8:8.2```

and for ubi-minimal : 

```registry.access.redhat.com/ubi8/ubi-minimal:8.2```